### PR TITLE
Edge 16 release

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

**Review by ljharb — APPROVED** [review]
Needs a rebase.

**Comment by mgol** [comment]
The Edge 16 data is out of date, the final version includes more features than currently advertised.

**Comment by mgol** [comment]
Actually, the only difference I see is that Edge 16 should have "arguments.caller removed" green in the "2017 misc" section. Could you flip that flag, @chicoxyzzy, when rebasing?

**Review by mgol — CHANGES_REQUESTED** [review]
Actually, the only difference I see is that Edge 16 should have "arguments.caller removed" green in the "2017 misc" section. Could you flip that flag, @chicoxyzzy, when rebasing?

**Comment by chicoxyzzy** [comment]
@mgol sure, I have no Edge 16 to test so I just bumped version. Flag is flipped now

**Comment by mgol** [comment]
> Flag is flipped now

I don't see any change in this PR that touches `data-es2016plus.js`. Are you sure you pushed the change?

**Comment by chicoxyzzy** [comment]
Oops wrong branch :) I have branches `edge16` and `edge16_release` locally

File: data-es2016plus.js [hunk 1]
```diff
@@ -2006,6 +2006,7 @@ exports.tests = [
      return tag`\01\1\xg\xAg\u0\u0g\u00g\u000g\u{g\u{0\u{110000}${0}\0`;
      */},
     res: {
+      edge16: true,
```
Comment (afmenez) [inline data-es2016plus.js:2009]: It seems you have updated the wrong test.

**Comment by afmenez** [comment]
Sorry, I opened a similar PR (#1203) because I haven't seen yours,  But I believe my results are more precise.

**Comment by chicoxyzzy** [comment]
@afmenez I'm ok with taking your PR. And I checked wrong test indeed. Lack of sleep is a bad thing :(

**Pull request closed without merging**